internal/pkg/template/env.go

```diff
@@ -135,6 +135,7 @@ type HTTPConfig struct {
 	ImportedCertARNs  []string
 	CustomALBSubnets  []string
 	ELBAccessLogs     *ELBAccessLogs
+	SecurityPolicy    *string
```

Comment:
It seems like we never assign value for this field. Ideally we should pass manifest value to this field otherwise it will always stay `nil`.

Reply:
In `internal/pkg/deploy/cloudformation/stack/env.go` we are passing manifest value to this field with the help of two functions that returns manifest value.
`func (e *EnvStackConfig) getPublicSecurityPolicy() *string `
`func (e *EnvStackConfig) getPrivateSecurityPolicy() *string`